fundingmanager.go

```diff
@@ -497,13 +520,68 @@ func (f *fundingManager) Start() error {
 				}
 				defer lnChannel.Stop()
 
-				err = f.sendChannelAnnouncement(channel, lnChannel,
-					shortChanID)
+				err = f.addToRouterGraph(channel, lnChannel,
+					shortChanID, private)
+				if err != nil {
+					fndgLog.Errorf("failed adding to "+
+						"router graph: %v", err)
+					return
+				}
+
+				if !private {
+					err = f.annAfterSixConfs(channel,
+						lnChannel, shortChanID)
+					if err != nil {
+						fndgLog.Errorf("error sending channel "+
+							"announcement: %v", err)
+						return
+					}
+				}
```

Comment:
Seems like the channel state can be deleted at this point if the channel is marked as private.